base/bootstrap.go

```diff
@@ -59,6 +64,41 @@ func NewCouchbaseCluster(server, username, password,
 	return &CouchbaseCluster{c: cluster}, nil
 }
 
+func (cc *CouchbaseCluster) GetConfigLocations() ([]string, error) {
+	if cc == nil {
+		return nil, errors.New("nil CouchbaseCluster")
+	}
+
+	buckets, err := cc.c.Buckets().GetAllBuckets(nil)
+	if err != nil {
+		return nil, err
+	}
+
+	bucketList := make([]string, len(buckets))
+	for bucketName := range buckets {
+		bucketList = append(bucketList, bucketName)
+	}
+
+	return bucketList, nil
+}
+
+func (cc *CouchbaseCluster) GetConfig(location, groupID string, valuePtr interface{}) (cas uint64, err error) {
+	if cc == nil {
+		return 0, errors.New("nil CouchbaseCluster")
+	}
+
+	res, err := cc.c.Bucket(location).DefaultCollection().Get(PersistentConfigPrefix+groupID, nil)
+	if err != nil {
```

Comment:
For other gocb functionality I've been checking for expected errors (e.g. not found) at this level, and returning the SG ErrNotFound to avoid leaking gocb imports out of base.